lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -617,10 +622,10 @@ public function get_stylesheet( $types = array( 'variables', 'styles', 'presets'
 	 * @return string Styles for the block.
 	 */
 	public function get_styles_for_block( $block_metadata ) {
-		$node = _wp_array_get( $this->theme_json, $block_metadata['path'], array() );
-
-		$selector = $block_metadata['selector'];
-		$settings = _wp_array_get( $this->theme_json, array( 'settings' ) );
+		$node             = _wp_array_get( $this->theme_json, $block_metadata['path'], array() );
+		$use_root_padding = _wp_array_get( $this->theme_json, array( 'settings', 'useRootPaddingAwareAlignments' ), false );
```

Comment:
Not a biggie, but maybe we could check for a boolean here?

```suggestion
		$use_root_padding = isset( $this->theme_json['settings']['useRootPaddingAwareAlignments'] ) && true === $this->theme_json['settings']['useRootPaddingAwareAlignments'];
```

These values also toggle the option:

```json
"useRootPaddingAwareAlignments": 10,
"useRootPaddingAwareAlignments": "lizard-people",
```

Reply:
Aw, where's the fun in that